payment/payment.go

```diff
@@ -3,15 +3,18 @@ package payment
 import (
 	"crypto/sha256"
 	"errors"
+	"fmt"
 	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/txscript"
 	"github.com/vulpemventures/go-elements/address"
 	"github.com/vulpemventures/go-elements/network"
 	"golang.org/x/crypto/ripemd160"
 	"hash"
 )
 
 const (
-	Op0 = 0x00
+	Op0             = 0x00 // 0
+	OpCheckMultiSig = 0xae // 174
 )
```

Comment:
We should use the opcodes from btcd instead of redefining them into payment https://github.com/btcsuite/btcd/blob/86fed781132ac890ee03e906e4ecd5d6fa180c64/txscript/opcode.go